### DISCUSSION TITLE
Status kirim sukses, tp tidak masuk

### DISCUSSION THREAD

**Discussion by shahib-mirbath** [kirimwa-id/forum#130] [Tanya Jawab]
Beberapa kali mendapati kirim WA status sukses, tp pesan tidak masuk. di WA device tidak ada pesan sama sekali, tapi di web WA ada pesan seperti gambar berikut:

![gambar](https://user-images.githubusercontent.com/46096681/185819752-e5d89cf6-efa6-45b8-b10f-de47a611fae3.png)

**Comment by kafaa-dev** [comment]
![image](https://user-images.githubusercontent.com/73120669/185866010-1c268202-3a2b-4643-b81a-a61becea3c83.png)
Saya menerima pesan dari pengguna handphone lain juga memang kadang begitu mas

Berdasarkan https://faq.whatsapp.com/2448118811981474/?locale=id_ID
perlu membuka WhatsApp di telepon yang digunakan untuk KirimWA.id

dan memastikan menggunakan WhatsApp versi terbaru

**Reply by shahib-mirbath** [reply]
@syahid246 saat saya lihat di WA device malah tidak ada pesan. Padahal ada pesan yg dikirim dan tampil di penerima.

WA device api.kirimwa.id dan penerima sudah yg paling baru.

Saya awalnya mengira karena bedanya versi, tp ternyata sama aja.

**Comment by valegilbert** [comment]
Sama, saya juga sampai sekarang banyak kasus seperti ini.. bingung apa yang harus dilakukan :-)
